data_juicer/utils/model_utils.py

```diff
@@ -255,13 +255,36 @@ def decompress_model(compressed_model_path):
     return diversity_model
 
 
+def prepare_diffusion_model(model_name_or_path):
+    """
+        Prepare and load an Diffusion model from HuggingFace.
+
+        :param model_name_or_path: input Diffusion model name or local
+            path to the model
+        :return: a Diffusion model.
+    """
+    import torch
+    from diffusers import StableDiffusionImg2ImgPipeline
+
+    # NOTE: It can be loaded as revision="fp16" and
+    # torch_dtype=torch.float16 when using cuda
+    model = StableDiffusionImg2ImgPipeline.from_pretrained(
+        model_name_or_path,
+        use_auth_token=True,
+        revision='fp32',
```

Comment:
Consider exposing these args to `prepare_diffusion_model`

Reply:
Please check [prepare diffusion model and test](https://github.com/alibaba/data-juicer/pull/200/commits/3d5125dfb7aafad5f7853d52a49f24eab5d2fdea)